app/com/arpnetworking/play/metrics/MetricsActionWrapper.java

```diff
@@ -62,17 +81,28 @@ public MetricsActionWrapper(final MetricsFactory metricsFactory) {
      * @param context Context of the HTTP request/response.
      * @return Name of the timer for the request/response.
      */
-    protected String createTimerName(final Http.Context context) {
-        // TODO(vkoskela): Use routes information to replace variable path parts with variable names. [MAI-280]
+    protected static String createRouteMetricName(final Http.Context context) {
         final Http.Request r = context.request();
         final StringBuilder metricNameBuilder = new StringBuilder("rest_service/");
         metricNameBuilder.append(r.method());
 
-        if (!Strings.isNullOrEmpty(r.path())) {
-            if (!r.path().startsWith("/")) {
+        final String route = context._requestHeader().tags().get("ROUTE_PATTERN").getOrElse(
+                new AbstractFunction0<String>() {
+                    @Override
+                    public String apply() {
+                        return "";
+                    }
+                })
+                .replaceAll("\\$([^<]+)<([^>]+)>", ":$1");
```

Comment:
Haha, really? It's going to be called that often ;) -- But yes, you are correct. Done.